backend/infrastructures/apps/load/views/serializers.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import serializers
+
+
+class ExtractDataSerializer(serializers.Serializer):
+    file_path = serializers.CharField(min_length=1)
+
+
+class OutputDataSerializer(serializers.Serializer):
```

Comment:
OutputDataReadSerializer

Reply:
Please add also something like this:

```python
class OutputDataBaseSerializer(serializers.Serializer):
    ....

class OutputDataReadSerializer(serializers.Serializer):
     full_name = serializers.CharField()
     age = serializers.IntegerField()
     is_satisfied = serializers.BooleanField()
```

Do the same for Extract serialzier

Reply:
I don't uderstand what you want to have in OutputDataBaseSerializer. Why do we need it ?

Reply:
It's just a good practise, to have easier way of extending in feature.